dev/app/builtin/stories/toggleFacetRefinement.stories.js

```diff
@@ -4,15 +4,15 @@ import { storiesOf } from 'dev-novel';
 import instantsearch from '../../../../index';
 import { wrapWithHits } from '../../utils/wrap-with-hits.js';
 
-const stories = storiesOf('ToggleRefinement');
+const stories = storiesOf('ToggleFacetRefinement');
```

Comment:
This is the name of the InstantSearch.js widget, not the helper method. We want to keep the name as is.